client/server.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express')
```

Comment:
Is this file needed to get your client running on AWS?

Reply:
Yes. This is necessary to run the production build on AWS instances